modules/snooker/src/main/ts/ephox/snooker/api/CopyRows.ts

```diff
@@ -11,13 +11,18 @@ const copyRows = (table: SugarElement<HTMLTableElement>, target: TargetSelection
   const warehouse = Warehouse.fromTable(table);
   const details = onCells(warehouse, target);
```

Comment:
Could this just be changed to `onUnlockedCells`? That'd prevent having to filter them out later and I believe it should achieve the same result.

Reply:
Actually, on second thought why are we filtering out the locked cells here? I thought the decision was to keep the cells when copying.

Reply:
Have also found a bug in relation to this logic. It doesn't appear to work if you right-click on the locked column and then click "Copy row". There's never any data added to our "fake" clipboard it seems.

Reply:
As discussed via slack, we're going to change the `CopyCols` logic to not allow copying locked cells either.

Also, the bug above was just an issue with my local environment it seems, so ignore that sorry.